homeassistant/util/yaml/loader.py

```diff
@@ -267,6 +267,10 @@ def _load_secret_yaml(secret_path: str) -> JSON_TYPE:
 def secret_yaml(loader: SafeLineLoader, node: yaml.nodes.Node) -> JSON_TYPE:
     """Load secrets and embed it into the configuration YAML."""
     secret_path = os.path.dirname(loader.name)
+    secret_filename = os.path.join(secret_path, SECRET_YAML)
```

Comment:
We can probably spare this variable by using something like 
```
 os.path.filename(loader.name) == SECRET_YAML
```

Reply:
Done